tests/foreman/cli/test_product.py

```diff
@@ -140,6 +140,7 @@ def test_positive_CRUD(self):
         })
         self.assertEqual(len(product['sync-plan-id']), 0)
         Product.delete({u'id': product['id']})
+        time.sleep(5)
```

Comment:
Could you please give a try to `call_entity_method_with_timeout` instead to directly using the `sleep` here ?